cmd/create/operatorroles/by_clusterkey.go

```diff
@@ -36,7 +36,7 @@ func handleOperatorRoleCreationByClusterKey(r *rosa.Runtime, env string,
 		}
 	}
 
-	if ocm.IsOidcConfigReusable(cluster) && len(missingRoles) == 0 {
```

Comment:
I'm a little bit confused, the flag says `forcePolicyCreation` but we skip `ValidateOperatorRolesMatchOidcProvider`, and inside `ValidateOperatorRolesMatchOidcProvider` we check if the policy is compatible, but we don't create policies.

Can you please elaborate on that?

Reply:
In the rush I ended up skipping more than wanted, adjusted so only the message and return are skipped